BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -99,6 +99,16 @@ public final class BlueprintView: UIView {
             invalidateIntrinsicContentSize()
         }
     }
+    
+    /// An optional explicit layout mode for this view. If `nil`, this view will inherit the layout
+    /// mode of its nearest ancestor, or use ``LayoutMode/default``.
+    public var layoutMode: LayoutMode? {
```

Comment:
Thinking outloud:
- Do we want to persist a previous "global" default value in user defaults? Eg imagining without persisting it somewhere, for a bit as the app starts up (or if you're logged out and we drive this from an account/status flag), we'll be using the "standard" value – which could also result in layouts being laid out one time with standard layout, and another time with SPL

Reply:
> Thinking outloud:
> 
> * Do we want to persist a previous "global" default value in user defaults? Eg imagining without persisting it somewhere, for a bit as the app starts up (or if you're logged out and we drive this from an account/status flag), we'll be using the "standard" value – which could also result in layouts being laid out one time with standard layout, and another time with SPL

Maybe. One reason I made the global a constant is that if you change it at runtime you need to invalidate _everything_, and there's currently no way to do that. We'd need every BlueprintView to listen to a global notification or something. It's much easier to set individual views to whatever value the app wants.